app/src/main/java/it/quip/android/actvitiy/CreateCircleActivity.java

```diff
@@ -86,8 +86,14 @@ public void done(ParseException e) {
     }
 
     private void finishWithResult() {
+        Circle createdCircle = circleHeaderFragment.getCircle();
         Intent data = new Intent();
-        data.putExtra(CREATED_CIRCLE, circleHeaderFragment.getCircle());
```

Comment:
This is unnecessary since we add the newly created circle object to the current user.